eZ/Publish/Core/MVC/Symfony/Routing/Generator/UrlAliasGenerator.php

```diff
@@ -102,8 +102,12 @@ public function doGenerate($location, array $parameters)
             $rootLocationId = $this->rootLocationId;
         }
 
+        $anchor = array_key_exists('_fragment', $parameters)
```

Comment:
Can _fragment contain other things then # anchor? If so, then name can be confusing.

Reply:
Fragment is probably better name than anchor: https://en.wikipedia.org/wiki/Fragment_identifier and it's also used by Symfony. I've just commited a variable name change: `$anchor -> $fragment`.